src/Entity/AbstractTranslatableEntity.php

```diff
@@ -2,20 +2,18 @@
 
 namespace AppBundle\Entity;
 
-use A2lix\I18nDoctrineBundle\Doctrine\ORM\Util\Translatable;
+use Doctrine\Common\Collections\ArrayCollection;
 use Doctrine\Common\Collections\Collection;
 use Symfony\Component\Validator\Constraints as Assert;
 
-trait EntityTranslatableTrait
+abstract class AbstractTranslatableEntity
```

Comment:
Why use an abstract class for this translatable behavior ?

I would prefer an interface (forcing implementation to defines several methods) + a trait for the `translate()` method (that would provide the method needed in the interface)

Why ? because using inheritance would force entities for this specific translation behavior.
(What if i want to translate CitizenActions and not Events?)

Reply:
@Remg , to able to replace `symfony/symfony` by independent packages `symfony/*` I should update 
`a2lix/I18nDoctrineBundle` to the release 0.2. The previous version (0.1) has `symfony/symfony` as the dependence and doesn't allow to remove our `symfony/symfony` package. Unfortunately, this new release (0.2) introduces a little BC break. 
I replaced by Abstract class, because our entity (Profile for example) that uses `EntityTranslatableTrait` which uses Translatable trait wasn't identified by the bundle's Listener as a Translatable Entity.